lib/manageiq/api/common/engine.rb

```diff
@@ -0,0 +1,9 @@
+module ManageIQ
```

Comment:
I think we need an inflection defined for ManageIQ as well as an initializer for it in the engine file.